core/src/test/resources/TableMetadataUnsupportedVersion.json

```diff
@@ -1,5 +1,5 @@
 {
-  "format-version": 4,
+  "format-version": 5,
```

Comment:
nit: maybe we should make this -50 or something ... Seems silly to keep bumping it by 1

Reply:
indeed, I've updated this to a large value > 0, since I actually wanted to test this with a positive unsupported version